Tests/UI/New UI/BillingForm/View/BillingFormButtonViewTests.swift

```diff
@@ -29,12 +29,6 @@ class BillingFormButtonViewTests: XCTestCase {
         XCTAssertEqual(view.image?.tintColor, style.button.disabledTintColor)
     }
 
-    func testButtonStyle() {
-        XCTAssertEqual(view.button?.isEnabled, style.button.isEnabled)
-        XCTAssertEqual(view.button?.tintColor, .clear)
-        XCTAssertEqual(view.button?.backgroundColor, .clear)
-    }
```

Comment:
Do we intentionally want to remove unit tests?

Reply:
Duplicate test cases